### DISCUSSION TITLE
QT Problem

### DISCUSSION THREAD

**Discussion by majdbousaad** [jeertmans/manim-slides#328] [Q&A]
I got this error:
```
could not load multimedia backend ""
QtMultimedia is not currently supported on this platform or compiler.
Failed to create QVideoSink "Not available"
Failed to initialize QMediaPlayer "Not available"
 ```
 
 Before this I had a problem that  ```we could not find qt  "windows" in ""```
 I then put an enviroment variable called QT_QPA_PLATFORM_PLUGIN_PATH with the value "pyside6/plugins/platforms"

**Comment by jeertmans** [comment]
Hello, did it fix your issue?

**Reply by majdbousaad** [reply]
Hello, I solved it by creating a venv enviroment.
But do you know what caused the problem outside the venv enviroment?

**Reply by jeertmans** [reply]
Probably a bad installation…

Did you try to make a clean install? Maybe take a look a pipx and install Manim Slides via pipx :)
